src/Controller/ShopwareFilesController.php

```diff
@@ -143,7 +151,13 @@ public function restoreShopwareFile(Request $request): JsonResponse
             opcache_reset();
         }
 
-        return new JsonResponse(['status' => sprintf('File at "%s" has been restored', $file)]);
+        $userName = $this->getUserName($context) ?? 'unknown';
+
+        $message = sprintf('File at "%s" has been restored by %s', $file, $userName);
```

Comment:
you can use parameter {name} and replace them with context see https://symfony.com/doc/current/logging.html

Reply:
Uhh, nice to know. But we use it for the result, too.